src/main/webapp/script.js

```diff
@@ -17,6 +17,31 @@ function onSignIn(googleUser) {
     'Welcome ' + profile.getName();
   document.getElementById('profileEmail').innerHTML = profile.getEmail();
   //   document.getElementById('profileId').value = profile.getId();
+
+  signInCallback(id_token);
+}
+
+function signInCallback(authResult) {
```

Comment:
As I mentioned in #23 this is simple as the code mentioned in the StackOverflow answer:
```js
var user = {
    'email': profile.getEmail(),
    'id': profile.getId(),
    // whatever other fields are needed
};

var formBody = [];
for (var property in details) {
  var encodedKey = encodeURIComponent(property);
  var encodedValue = encodeURIComponent(details[property]);
  formBody.push(encodedKey + "=" + encodedValue);
}
formBody = formBody.join("&");

fetch('/users', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
  },
  body: formBody
})
```

Though actually I think this might work too:
```js
var user = {
    'email': profile.getEmail(),
    'id': profile.getId(),
    // whatever other fields are needed
};

fetch('/users', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
  },
  body: new URLSearchParams(user).toString(),
})

```